### PR TITLE
Gcmon 435

### PR DESCRIPTION
**Pull request by kmschoep-usgs**

### CODE REVIEW

File: gcmrc-ui/src/main/webapp/pages/reachview/onReady.js [hunk 1]
```diff
@@ -18,10 +18,17 @@ $(document).ready(function onReady() {
 	});
 	
 	GCMRC.Page.buildPORView($('#porContainer'), GCMRC.Page.earliestPositionISO, GCMRC.Page.latestPositionISO);
-
-	var bedLoadList = [GCMRC.Page.sliderConfig.bedLoad];
+        
+        if ("DINO" === CONFIG.networkName) {
```
Comment (ee-usgs) [inline gcmrc-ui/src/main/webapp/pages/reachview/onReady.js]: There is a lot of special ‘DINO’ checking.  I know a lot already existed, but one way to centralize this a bit would be to add a method like this at  /app/gcmrc.jsp:77
```javascript
GCMRC.isDinoNetwork = function(netName) {
  return netName != null && 'DINO' == netName.toUpperCase();
}
```
As far as I can tell, that is loaded at the top of every page and would centralize how we detect the DINO network w/o having to have ‘DINO’ strings all over the place.

File: gcmrc-ui/src/main/webapp/app/graphing/graphing.js [hunk 2]
```diff
@@ -134,9 +134,13 @@ GCMRC.Graphing = function(hoursOffset) {
 		var timeColumn = "time";
 		var conf = $.extend({}, config);
 		var hasData = false;
+		var bedLoadData;
+		if (graphToMake.groupId === 'sandbudget') {
```
Comment (thongsav-usgs) [inline gcmrc-ui/src/main/webapp/app/graphing/graphing.js]: Not a comment about what you did but the code base in general, we have way too many uber functions with branching logic for various cases. Might need to put in a ticket to tackle this once and for all.

File: gcmrc-ui/src/main/webapp/pages/reachview/page.js [hunk 3]
```diff
@@ -502,6 +505,60 @@ GCMRC.Page = {
 				GCMRC.Page.drawBudget(config);
 			};
 		};
+		
+		var BedloadCoeff = function() {
```
Comment (thongsav-usgs) [inline gcmrc-ui/src/main/webapp/pages/reachview/page.js]: can we add a verb to this function name? is it a load/get/calc?

File: gcmrc-ui/src/main/webapp/pages/reachview/page.js [hunk 4]
```diff
@@ -573,6 +635,43 @@ GCMRC.Page = {
 
 		container.append(result.join(""));
 	},
+	toggleChange: function(useBedload){
```
Comment (thongsav-usgs) [inline gcmrc-ui/src/main/webapp/pages/reachview/page.js]: Is there a more explicit name we can use? This implies a global toggle, I think it's just the bedload toggle?

**Pull request merged**